stratum/hal/lib/tdi/tdi_sde_wrapper.cc

```diff
@@ -0,0 +1,3817 @@
+// Copyright 2019-present Barefoot Networks, Inc.
```

Comment:
The rest of the comments for this file are tofino vs dpdk. So I will not add any more and just leave the understanding that we will need to figure this out.